### DISCUSSION TITLE
The new user interface is inconsistent with the Readme documentation

### DISCUSSION THREAD

**Discussion by Nothend** [d8ahazard/sd_dreambooth_extension#1403] [General]
The new user interface is inconsistent with the Readme documentation,There are very few tutorials on the Internet based on the new operating interface, which causes a lot of trouble.Here are some screenshots:

1. On the `Creat` page .
    
![image](https://github.com/d8ahazard/sd_dreambooth_extension/assets/73117056/7613c1da-6c85-4f96-a0a5-7eb905d8bb4e)

in the new version , there are serval types of model

![image](https://github.com/d8ahazard/sd_dreambooth_extension/assets/73117056/51b47b12-308f-4cd6-91dd-56083a845be3)

2. There is no Performance Wizard (WIP)  in new version 

    
![image](https://github.com/d8ahazard/sd_dreambooth_extension/assets/73117056/1a0b2686-e8aa-497e-92ef-dfa7345e8653)

![image](https://github.com/d8ahazard/sd_dreambooth_extension/assets/73117056/0c23473e-95db-4918-bf37-ec42d608ef98)

The Readme shoud change as well

**Comment by d8ahazard** [comment]
Sorry, I'm aware. My time is at a minimum, and I'm using what I have to do
a pretty substantial rewrite, after which time I'll be updating the
documentation. I'm always open to contributions in the meanwhile, and on
discord if you have and direct questions.

My apologies for the confusion during this transition period.

**Reply by Nothend** [reply]
appreciate it !!!
